src/helpers.rs

```diff
@@ -54,6 +61,43 @@ pub fn copy_dir(source: &Path, dest: &Path) -> Result<()> {
     Ok(())
 }
 
+/// Find the root zine file in current dir
+fn zine_file_existed(path: &PathBuf) -> bool {
```

Comment:
Generic to `Path` (`<P: AsRef<Path>>`) is better than the concrete `PathBuf` which is an owned path.